doc/releases/changelog-0.9.0.md

```diff
@@ -88,8 +71,8 @@
 
   @qjit
   def fn(x):
-      circuit_pipeline = pipeline(my_pipeline)(circuit)
-      circuit_other = pipeline(my_other_pipeline)(circuit)
+      circuit_pipeline = pipeline(pass_pipeline=my_pipeline)(circuit)
```

Comment:
Is it needed or was it not clear enough?

Reply:
it was because of this: https://github.com/PennyLaneAI/catalyst/pull/1239#discussion_r1820345097

Reply:
This is now fixed, and the `(pass_pipeline=...)` is not needed.